scripts/eurostat/regional_statistics_by_nuts/fertility_rate_mother_age/README.md

```diff
@@ -0,0 +1,94 @@
+# Importing EuroStat Education Attainment Into Data Commons
+Author: eftekhari-mhs
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Artifacts](#import-artifacts)
+1. [Generating Artifacts](#generating-artifacts)
+
+## About the Dataset
+
+### Download URL
+
+[TSV] file is available for [download](https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/demo_r_find3.tsv.gz).
+
+[Data explorer](https://appsso.eurostat.ec.europa.eu/nui/show.do?dataset=demo_r_find3&lang=en)
+
+### Overview
+
+This dataset has the fertility rate, median and mean age of mother at childbirth for European Union (EU) countries to their NUTS3 geos over [2013-2018]. Codes used in the dataset: 
+
+- TOTFERRT – Total fertility rate.
+- AGEMOTH – Mean age of women at childbirth.
+- MEDAGEMOTH – Median age of women at childbirth.
+
+
+The original dataset is broken up into 3 major families of variables:
+1. Date: Years from 2013 to 2018 with some holes in time series marked with note "b" in the original dataset
+2. indic_de,unit,geo: e.g. AGEMOTH,YR,AL	 (Mean age of women at childbirth, Years, nuts/AL)
+
+indic_de,unit,geo is further broken down into:
+1. indic_de: 'TOTFERRT', 'AGEMOTH', 'MEDAGEMOTH'
+2. unit: 'YR' (age), 'NR' (fertility rate)
+3. geo: NUTS3 codes for regions of Europe
+
+The cleaned csv is broken up into the following major families of variables:
+1. Date: Subset of years from 2000 to 2019 with values in the original dataset

```

Comment:
I am confused how the original dataset has data from 2013 to 2018 and the final CSV has data from 2000 to 2019.

Also, does this mean the dataset is no longer being updated?

Reply:
This is a typo, it should be 2013-2018. Thanks for catching it!